modules/ocf/manifests/networking.pp

```diff
@@ -54,7 +58,7 @@
         # necessarily correct, falling back will help make sure VMs upgraded
         # from jessie have networking configured when they reboot, since they
         # retain the old 'eth0' interface name until after reboot.
-        # XXX find out how to predict new interface names
+        # XXX: find out how to predict new interface names
```

Comment:
this is probably better as a TODO anyway, whereas XXX is just sort of a general "beware, here there be dragons"-type comment